src/widgets/input.css

```diff
@@ -103,9 +126,18 @@ input::placeholder {
   font-size: 13px;
 }
 
+.password button {
+  position: absolute;
+  right: 0;
+  background: rgba(255, 255, 255, 1.0);
+}
+
+.password-text button::before,
 .password button::before {
+  cursor: pointer;
   background-repeat: no-repeat;
   background-size: 16px 16px;
+  background-position: 0 2px;
```

Comment:
This is kind of arbitrary, but seems to get the show/hide icon better aligned with the dots and midpoint of text. Might be worth yanking out.